style.css

```diff
@@ -1,595 +1,601 @@
-  width: 327px;
-  height: 72px;
-}
-
-.list-container img {
-  margin-left: 20px;
-}
-
-.languages-list {
-  width: 327px;
-  height: 248px;
-  padding: 0;
-  display: flex;
-  flex-direction: column;
-}
-
-.language {
-  display: flex;
-  align-items: center;
-  background-color: #f7f7f9;
-  width: 300px;
-  min-height: 66px;
-  height: 66px;
-  box-sizing: border-box;
-  margin-top: 12px;
-}
-
-.language p {
-  margin-left: 12px;
-}
-
-.language img {
-  margin-left: 12px;
-}
-
-.list-container p {
-  font-family: 'Poppins', Arial, Helvetica, sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 15px;
-  color: #253858;
-  width: 237px;
-  height: 24px;
-}
-
-.options-list2 .list-container {
-  border-bottom-width: 3px;
-  border-bottom-color: #f7f7f9;
-  border-bottom-style: solid;
-}
-
-.contact-form {
-  width: 375px;
-  height: 779px;
-  background: url(images/form-background.svg) no-repeat;
-  background-size: 375px 779px;
-  display: flex;
-  flex-direction: column;
-  padding-left: 24px;
-  box-sizing: border-box;
-  margin-top: 114px;
-}
-
-.contact-form h2 {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  color: #fff;
-  text-align: center;
-  width: 327px;
-  margin-top: 103px;
-  margin-bottom: 12px;
-}
-
-.contact-form p {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 28px;
-  color: #ebebff;
-  text-align: center;
-  width: 327px;
-  height: 140px;
-  margin-top: 0;
-  margin-bottom: 50px;
-}
-
-.contact-form input {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  align-self: center;
-  width: 327px;
-  height: 48px;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  margin-bottom: 18px;
-  padding: 15px 38px 15px 16px;
-  box-sizing: border-box;
-}
-
-.contact-form textarea {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: #b3bac5;
-  width: 327px;
-  height: 170px;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  margin-bottom: 22px;
-  padding: 15px 38px 15px 16px;
-  box-sizing: border-box;
-}
-
-.contact-form button {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  color: #6070ff;
-  height: 48px;
-  width: 140px;
-  background: #fff;
-  border-radius: 8px;
-  padding: 12px 16px;
-  align-content: center;
-  border-style: none;
-}
-
-.contact-form button:hover {
-  background-color: lightblue;
-  transition: 0.25s;
-}
-
-.contact-form button:active {
-  background-color: lightgrey;
-  border-color: grey;
-}
-
-.line {
-  height: 5px;
-  width: 134px;
-  border-radius: 5px;
-  background-color: black;
-  margin-top: 32px;
-  margin-left: 96px;
+@media (max-width: 480px) {
+  .desktop {
```

Comment:
- [ ] According to the project requirements, I just noticed that the media queries for the Desktop breakpoint in the CSS file are supposed to have the value for the `min-width` property to be in the range of `768px `or `wider`) and not `480px`. Make sure you fix this to match the project requirements.